src/Slot-Examples/BooleanSlot.class.st

```diff
@@ -76,6 +76,24 @@ BooleanSlot >> installingIn: aClass [
 	offset := self calculateOffset: aClass.
 ]
 
+{ #category : #testing }
+BooleanSlot >> isAccessedIn: aMethod [
+	(baseSlot isAccessedIn: aMethod) ifFalse: [ ^false ].
+	^super isAccessedIn: aMethod
```

Comment:
I see it is same methods in PropertySlot.
What about extracting common superclass? like DerivedSlot?

Reply:
Yes, I was thinking about this... I will add a note to my TODO